services/core-web/common/actionCreators/projectActionCreator.ts

```diff
@@ -615,7 +615,7 @@ export const postNewDocumentVersion = ({
     )
     .then((response: AxiosResponse<IProjectSummary>) => {
       notification.success({
-        message: "Successfully sent new document version",
+        message: "Successfully created new document version",
```

Comment:
I loooooove how generic this function looks. Is it generic enough to put it in a documentActionCreator file?

Reply:
Good idea!  I'll take a look at adding that on Monday!